### ISSUE TITLE
STATIC 15 with Gnome freezes at login screen

### ISSUE THREAD

**Issue by minikN** [geckolinux/geckolinux-project#72]
Hello, unfortunately I can't say much more than the title already says. I installed it yesterday for the first time. After the first boot it freezed on the login screen (I disabled automatic login in the installer).

Because I manually partitioned my drive (and it was late) I just did a fresh try today. After the first startup it didn't freeze (yeah). Than all I did was an update via YaST. Rebooted. Freeze on Login screen again.

Hardware:
Dell Optiplex 7010
i5-3550
8 GB Ram
Intenso 256 GB SSD

Cheers mate.

**Comment by minikN** [comment]
Issue can be closed. I think I also had [this](https://github.com/geckolinux/geckolinux-project/issues/60) issue. However, I couldn't fix it even after following the conversation.

If I updated the system like you described it works. However I also get a kernel conflict. But even after that, the "online update" option shows me 5 or so security updates. I also see [this](https://github.com/geckolinux/geckolinux-project/issues/60#issuecomment-401853884). But doing the same didn't work. System freezes after reboot.

I've never had this problem with the official version. So I'm going back. Please don't take this offensively. Good luck with the project!

**Comment by geckolinux** [comment]
Hi there, thanks for reporting this. What brand of graphics hardware do you have?

**Comment by minikN** [comment]
GeForce GTX 1050 Ti.

**Comment by minikN** [comment]
Hello,
After my second comment I was more or less done with the matter. I switched back to openSuSE and was fine with it. However I actually wanna give this a second try because I like the idea of the project. So here we go, another, more detailed explanation:

I prepared a USB containing the stable Gnome image with `dd` and booted from it.
Then I start the Calamares Installer, choose my language and continue. At the partitioning I choose my target drive and select the third option (wipe drive). I choose not to automatically log in an fire off the installation.
BTW: I use two monitors, if I boot into the live usb with both monitors turned on, I won't be able to see any mouse pointer. I need to shut down my system. Turn off one monitor and boot it up again. Then it display the mouse pointer correctly. However this also happens with the original version when launching the installer.

After that I boot from the hdd and see the login screen. I type in user and pw, hit enter. Then it appears to be loading the desktop for about two seconds, then jumps back to the login screen (only displaying my name) and freezes. I can't really say if it froze or not. However I can't do anything, it won't accept any input. So I have to hard reset.

I had an idea that it might be my dual monitor setup. I completely unplugged one tried, then the other one. No success. It's not really a freeze because after logging in, I'm able to use it for a couple of seconds, then it throws me back to the login screen and then freezes. It's really weird. 

Hope this helps in diagnosing.

**Comment by minikN** [comment]
Update: I managed to get into the OS. I did it by choosing the advanced option in the bootloader menu. The system doesn't freeze if I use the one of the kernel options there. Now, when I try to update the system like you explained in the docs, I get this message

![screenshot from 2018-09-17 23-19-51](https://user-images.githubusercontent.com/1888338/45645198-067ad300-bac0-11e8-8510-1753f3dcd03e.png)

It is in German (I do not know why, I have not installed any other language yet). But hopefully you can decipher it, it is a conflict between `kernel-default-base-4.12.14-lp150.12.16.1.x86_64` and `kernel-default-4.12.14-lp150.12.16.1.x86_64`.

**Comment by minikN** [comment]
Okay, another update (and sorry for the spam btw): I decided to do another reinstall, but this time instead auto partitioning the disks automatically, I did my usual setup with btrfs. I manually created one ESP with 300 MB FAT32, one swap with 8 GB and filled the rest with root BTRFS. Installation went fine. Rebooted, No freeze. However it froze again after the second boot. Maybe this helps you?

![image](https://user-images.githubusercontent.com/1888338/45647662-b18e8b00-bac6-11e8-9566-16f799665b12.png)

Cheers.

**Comment by minikN** [comment]
Last update for today (promise): I can't really find any consistancy in this. Now it's also freezing no matter what boot option I choose. Seems to be random, I don't know. I noticed a nouveau related message during boot: `Nouveau [0000:01:00,0] DRM: Pointer to flat panel table invalid`

And after some hours of fiddling, this seems to be the issue.
https://wiki.archlinux.org/index.php/nouveau#Flat_Panel_Table_Invalid

**Comment by geckolinux** [comment]
Thanks a lot for the additional testing. Could you please try uninstalling the `xf86-video-nv` and `xf86-video-nouveau` driver packages? This should make it fallback to the Xorg xmodesetting driver, which tends to be more reliable and still should give decent performance.

**Comment by geckolinux** [comment]
Also, you can safely remove the `kernel-default-base` package (**NOT** `kernel-default`!!). In fact it might be best to try that before you play with the Xorg modesetting driver.

**Comment by minikN** [comment]
Hello. Thanks for your suggestion. I'm a couple of step further to victory now, but not quite.

First, in order to successfully get into Gnome again, I had to temporarily blacklist nouveau. I did that by adding `modprobe.blacklist=nouveau` to my kernel parameters. Once in Gnome, I first updated my whole system (given the method you provide). I removed `kernel-default-base`. After that I rebooted, sadly, it didn't fix the bug.
Blacklist nouveau again, boot into gnome and removed `xf86-video-nv` and `xf86-video-nouveau`.

However, that didn't do much. The nouveau kernel module was still being loaded upon booting. So I decided to perma-blacklist it by creating the file `/etc/modprobe.d/blacklist-nouveau.conf` and adding
    
    blacklist nouveau
    options nouveau modeset=0

and re-creating the ramdisk with `mkinitrd`. After that it didn't get loaded anymore, although the module is still being shown in `hwinfo` under `Display -> Nvidia GeForce GTX 1050 Ti -> Drivers` (Sorry, I can't post a picture at the moment, I am at work).

So after that, I decided to install the proprietary nvidia drivers from the community repo. Through YaST I installed the `x11-video-nvidiaG04` package. It automatically installed two other packages alongside it.

I always did that in openSuSE and it always worked. However after rebooting, still no nvidia drivers loaded. I opened `Nvidia X Server Settings` and a message popped up saying I'm not using the nvidia driver because they haven't been configured for X. I auto-created a conf file with `nvidia-xconfig` in `/etc/X11/xorg.conf` and rebooted.

Unfortunately that didn't work at all. X would not start and I would just be welcomed with a console login-prompt. Beautiful yeah, but not what I was hoping to see.

I removed the conf file again and am back to default Xorg drivers.

This is where I am now.

Cheers.
